src/renderer/components/update-button/update-button.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import styles from "./styles.module.scss";
+
+import React, { useState } from "react";
+import { Menu, MenuItem } from "../menu";
+import uniqueId from "lodash/uniqueId";
+import { noop } from "lodash";
+import { cssNames } from "../../utils";
+
+interface UpdateButtonProps {
+  warningLevel?: "light" | "medium" | "high";
+  update: () => void;
```

Comment:
Maybe it should be optional and you won't need things like this?

```tsx
<UpdateButton update={noop} /> // top-bar.tsx
```

Reply:
This is intentional. Without `update` function this component doesn't make any sense. Current form with `update={noop}` is used only temporary before business logic for this button will be ready.